### PR TITLE
Add K8s/GitHub Actions chapter

### PR DESCRIPTION
**Pull request by jonatasbaldin**
## Description:

Adds K8s/GitHub Actions chapter.

## I solemnly swear that:

- [x] I ran the hugo server and looked at my changed in the browser with my own eyes
- [x] I ran the linter and there were no errors
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: content/k8s/github-actions/backend/_index.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  - k8s/github-actions/frontend
+ready: true
+submission_type: continue_repo
+tags:
+- kubernetes
+- github
+title: Backend Pipeline
+---
+
+Now that you understand how a workflows works, create a file named `python.yaml` under `.github/workflows` to build and push the backend Docker image.
```
Comment (t-witchy) [inline content/k8s/github-actions/backend/_index.md:19]: This feels too light for a single card

File: content/k8s/github-actions/helm/_index.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  build-and-push:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v2
+
+    - name: Build and Push Helm Chart
+      env:
+        DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
```
Comment (t-witchy) [inline content/k8s/github-actions/helm/_index.md:41]: can we consider removing these lines and asking the learners to ensure the secrets pull through.

**Review by t-witchy — COMMENTED** [review]
This feels too easy

**Comment by jonatasbaldin** [comment]
hey @t-witchy, thanks for the review!

> This feels too easy

What do you envision as more challenging?

I'll pull @RLutsch here as well to discuss that.

**Review by RLutsch — APPROVED** [review]
lgtm

**Review by t-witchy — APPROVED** [review]

**Pull request merged**